ws-nextjs-app/pages/[service]/send/[id]/SuccessMessage/index.tsx

```diff
@@ -0,0 +1,101 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import Heading from '#app/components/Heading';
+import Paragraph from '#app/components/Paragraph';
+import { useContext } from 'react';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import styles from './index.styles';
+
+const DEFAULT_RETENTION_POLICY_DAY = '270';
+const DEFAULT_EMAIL = 'CannotFindEmail@bbc.co.uk';
+
+const defaultTranslations = {
+  confirmationStepTitle: 'Message sent',
+  confirmationStepDescriptionHtml: 'Thanks for getting in touch.',
+  referenceNumber: 'Reference number',
+  submissionInfoSignedOutMessage:
+    'You may wish to make a note of these details for your reference.',
+  retentionPeriodDays:
+    "We'll keep your submission for up to {{days}} days – and if we don't use it we'll then delete it and any other information you sent us.",
+  privacyInfoHtml:
+    "Don't worry, we protect your information — read the {{privacyInfoLink}} for more details.",
+  emailToHtml:
+    "If you change your mind and don't want us to use it, just email us at {{emailLink}}. Don't forget the reference number.",
+  removalGuidelineText:
+    'If you submitted something for a programme or online, we won’t be able to remove it once we use it.',
+};
+
+const defaultRetentionPolicy = {
+  href: 'https://www.bbc.com/usingthebbc/privacy/',
+  text: 'Privacy Policy',
+};
+
+const SucccessMessage = ({ submissionId }: { submissionId: string | null }) => {
```

Comment:
Could we access `submissionId` from `useFormContext()` rather than passing it as a prop?